image_generation/image_generation.py

```diff
@@ -9,15 +9,20 @@
 from huggingface_hub import hf_hub_download
 import gc
 
+_pipeline = None
+_loading_complete = False
+_loading_thread = None
+
+
 def flush():
+    global _pipeline
```

Comment:
The key is NOT offloading the model to CPU (commenting out `pipe.enable_model_cpu_offload()`?) and dropping the pipeline when flushing.